src/server/components/lists/searches/translators-interpreters.ts

```diff
@@ -29,12 +30,30 @@ export const translatorsInterpretersQuestionsSequence = [
   QuestionName.readDisclaimer,
 ];
 
+const serviceTypeToNoun: {[key: string]: string} = {
```

Comment:
Out of curiosity, why did you not use `Record<string, string>` here?

Reply:
typically I would use `Record` for advanced type usage, when mapping types to other types. I also don't _agree_ with tsc on this but ¯\_(ツ)_/¯

eg
```.ts
type ServiceTypes = "Translators" | "Interpreters";
const serviceTypeToNoun: Record<ServiceType, string> = {
  // must implement all members of ServiceType
}
```

I would've done the above, but there is a possibility of query param poisoning (i.e. unknown string), so I can't use `Record<ServiceType, ...>` (otherwise ts will complain that "string" doesn't conform with ServiceType). 

Record, when you expand it, is a generic that does this 
```
type Record<K extends keyof any, T> = {
    [P in K]: T;
};
```
and `[P in string]: string`  does not make any sense to me, and is not my intention